src/status_im/wallet/core.cljs

```diff
@@ -542,10 +519,12 @@
                                  {:to    to-norm
                                   :value amount-hex}
                                  {:to       (ethereum/normalized-hex address)
-                                  :from     from-address
                                   :data     (abi-spec/encode
                                              "transfer(address,uint256)"
-                                             [to-norm amount-hex])}))}))))
+                                             [to-norm amount-hex])
+                                  ;;Note: data from qr (eip681)
```

Comment:
these two should be in the first map that is merged because they should be added in both cases

Reply:
mhm, probably you're right, thank

Reply:
@yenda  done